src/components/10-atoms/checkbox/index.react.d.ts

```diff
@@ -8,6 +8,8 @@ interface AXACheckboxProps {
   disabled?: boolean;
   error?: string;
   onChange?: (value: any) => void;
+  onFocus?: () => void;
```

Comment:
@MarekLacoAXA do you want value: any here, too?

Reply:
Why any? Can't it be `Function` or something like that?